### DISCUSSION TITLE
Dotnet bindings

### DISCUSSION THREAD

**Discussion by ptr1120** [cvc5/cvc5#7247] [Q&A] [answered]
Hello,

this is more question instead of a bug report. I am interested in using Cvc in Dontnet (c#). As I see you already have bindings for Python and Java. Are there any plans to support usage of Cvc on Dotnet?

Best regards 
Peter

**Comment by 4tXJ7f** [comment, marked as answer]
Thank you for your request. Out of curiosity: What is your use case? We currently have no immediate plans for implementing .NET bindings. Java and Python bindings have been the most frequently requested ones and we unfortunately have limited bandwidth to implement additional bindings. If someone were to contribute .NET bindings, we would be happy to include them, though.
